armbian/base/config/ssh/authorized_keys.readme

```diff
@@ -0,0 +1,4 @@
+# For secure SSH login, add your SSH key to the file `authorized_keys` in this directory.
+# It is excluded from Git versioning control.
+#
+# See this introduction for additional information: <https://www.atlassian.com/git/tutorials/git-ssh>
```

Comment:
Why not simply add the contents of this file above `BASE_ADD_SSH_KEYS` option in build.conf?
This file is much harder to discover compared to reading the doc comment in the config file.

Reply:
I thougt about that but think this has its limits when working with multiple keys. Would work technically, I guess, but I'm not really comfortable with multi-line variables.

Reply:
Multi-line variables? What I meant is this, in the `build.conf` file:

```
# Add SSH keys in ./config/ssh/authorized_keys file for remote login.
# The file is excluded from Git versioning control.
# See this introduction for additional information: <https://www.atlassian.com/git/tutorials/git-ssh>
#BASE_ADD_SSH_KEYS="false"
```

Reply:
Ah, I read that wrongly and thought about adding the keys themselves to the build.conf. Good point, will add the info here.

Reply:
User can now add own keys in `armbian/base/config/authorized_keys`, instructions are moved to `build.conf`